networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/discovery/discv5/NodeRecordConverter.java

```diff
@@ -36,8 +36,10 @@ public class NodeRecordConverter {
 
   public Optional<DiscoveryPeer> convertToDiscoveryPeer(
       final NodeRecord nodeRecord, final SchemaDefinitions schemaDefinitions) {
+    // TODO: https://github.com/Consensys/teku/issues/8069
```

Comment:
Not clear why it's needed here
We get tcp6 in `or` here, isn't it not enough? We need list here too?

Reply:
to be fair, probably `or` is enough. It's just that in this case, if we are dual-stack, we would always try to connect to IPv4 when ideally we would prefer IPv6

Reply:
Will tackle in future PR

Reply:
So my message was in case if someone other will take it in future, I want it to be clear for anyone else, what do we want there ideally